### PR TITLE
Checkout UI changes

### PR DESCRIPTION
**Pull request by rajasekarcbt**
1. checkout
2. profile
3. profile update
4. password update

On the above page, I added page designs and mobile responsiveness.

user-pro1
user-pro2
user-pro3
user-pro4
user-pro5

### CODE REVIEW

File: backend/frontend/src/css/checkoutscreen.css
Comment (TharunCBT) [inline backend/frontend/src/css/checkoutscreen.css:1]: billing-cage .card is repeated check it

File: backend/frontend/src/css/checkoutscreen.css [hunk 1]
```diff
@@ -1,162 +1,69 @@
...
 }
 
-@media (max-width: 770px) {
-    .checkout-content, label {
-        padding: 0;
-        margin: 0;
-    }
+.checkout-card button {
+    width: 100% !important;
 }
```
Comment (TharunCBT) [inline backend/frontend/src/css/checkoutscreen.css:69]: Nit - can add checkout-card button inside checkout-card?

File: backend/frontend/src/css/profilescreen.css [hunk 2]
```diff
@@ -31,4 +12,108 @@
...
+
+.badge.bg-danger {
+    background-color: rgb(255, 0, 0, 0.3) !important;
+    color: #d23a12;
+}
+
+.badge.bg-completed {
+    background-color: rgba(27, 179, 51, 0.2) !important;
+    color: #097E52;
+}
```
Comment (TharunCBT) [inline backend/frontend/src/css/profilescreen.css:48]: NIT: badge.bg can added inside badge?

File: backend/frontend/src/css/updateprofilescreen.css [hunk 3]
```diff
@@ -267,4 +161,140 @@ form.otp-form {
...
+}
+
+.update-userprofile button:hover {
+  background: #097E52 !important;
+  border-color: #097E52 !important;
+}
+
+.update-userprofile button span {
+  margin-right: 5px;
+}
```
Comment (TharunCBT) [inline backend/frontend/src/css/updateprofilescreen.css:236]: NIT: same has the above changes can add update-userprofile related css inside it's parent class.

File: backend/frontend/src/screens/ProfileScreen.js [hunk 4]
```diff
@@ -117,216 +119,294 @@ function ProfileScreen() {
...
+                                  </span>
+                                </TableCell>
+                                <TableCell className="text-pink">
+                                  <a
+                                    onClick={() => redirectBooking(booking.id)}
+                                  >
+                                    <Eye
+                                      size={18}
+                                      color="#1859e5"
+                                      className="eye-icon"
```
Comment (TharunCBT) [inline backend/frontend/src/screens/ProfileScreen.js:326]: NIT: can add this inline css for eye-icon in the profilescreen.css

**Review by TharunCBT — APPROVED** [review]

File: backend/frontend/src/css/checkoutscreen.css
Comment (rajasekarcbt) [reply backend/frontend/src/css/checkoutscreen.css:1]: I changed it

**Pull request merged**